test/distributed/org/apache/cassandra/distributed/test/accord/AccordIntegrationTest.java

```diff
@@ -72,17 +80,29 @@ private static void assertRow(Cluster cluster, String query, int k, int c, int v
                                                    .build());
     }
 
-    @Test
-    public void testQuery() throws Throwable
+    private static void test(Consumer<Cluster> fn) throws IOException
     {
-        String keyspace = "ks" + System.currentTimeMillis();
-        try (Cluster cluster = init(Cluster.build(2).withConfig(c -> c.set("write_request_timeout_in_ms", TimeUnit.MINUTES.toMillis(10))).start()))
+        try (Cluster cluster = createCluster())
         {
             cluster.schemaChange("CREATE KEYSPACE " + keyspace + " WITH REPLICATION={'class':'SimpleStrategy', 'replication_factor': 2}");
             cluster.schemaChange("CREATE TABLE " + keyspace + ".tbl (k int, c int, v int, primary key (k, c))");
             cluster.forEach(node -> node.runOnInstance(() -> AccordService.instance.createEpochFromConfigUnsafe()));
             cluster.forEach(node -> node.runOnInstance(() -> AccordService.instance.setCacheSize(0)));
 
+            fn.accept(cluster);
+        }
+    }
+
+    private static Cluster createCluster() throws IOException
+    {
+        // need to up the timeout else tests get flaky
+        return init(Cluster.build(2).withConfig(c -> c.set("write_request_timeout_in_ms", TimeUnit.SECONDS.toMillis(10))).start());
```

Comment:
I found tests were flaky due to timeouts, so bumping to 10s made them more stable